xunit.runner.visualstudio.desktop/Sinks/VsDiscoverySink.cs

```diff
@@ -71,14 +71,18 @@ public void Dispose()
             discoveryEventSink.Dispose();
         }
 
-        public static TestCase CreateVsTestCase(string source, ITestFrameworkDiscoverer discoverer, ITestCase xunitTestCase, bool forceUniqueName, LoggerHelper logger, bool designMode)
+        public static TestCase CreateVsTestCase(string source, ITestFrameworkDiscoverer discoverer, ITestCase xunitTestCase, bool forceUniqueName, LoggerHelper logger, TestPlatformContext testPlatformContext)
         {
             try
             {
-                var serializedTestCase = discoverer.Serialize(xunitTestCase);
                 var fqTestMethodName = $"{xunitTestCase.TestMethod.TestClass.Class.Name}.{xunitTestCase.TestMethod.Method.Name}";
                 var result = new TestCase(fqTestMethodName, uri, source) { DisplayName = Escape(xunitTestCase.DisplayName) };
-                result.SetPropertyValue(VsTestRunner.SerializedTestCaseProperty, serializedTestCase);
```

Comment:
Issue: XunitTestCase property is required only for scenarios where VSTest can send a TestCase object later (e.g. Discovery in IDE, followed by Run Selected tests). A command line test run is a fire-and-forget operation, test property need not be serialized in this case.

Solution: add a `RequireXunitTestProperty` flag to control this.